server/controllers/article.js

```diff
@@ -2,9 +2,11 @@ import { Op } from "sequelize";
 import isEmpty from "lodash.isempty";
 
 import models from "@models";
+import { getBaseUrl } from "@helpers/users";
 import { httpResponse, serverError } from "@helpers/http";
 import { pagination } from "@helpers/pagination";
 import { generateUniqueSlug, calculateReadTime } from "@helpers/articles";
+import { sendPublishedArticleNotification } from "./notification";
```

Comment:
can you use our module resolver please

Reply:
Module resolver does not affect files that are in the same folder. You can take a look in the `index.js` file in the middlewares folder.